learn_django/trip_project/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+class Trip(models.Model):
+    trip_title = models.CharField(max_length=60)
+    trip_description = models.TextField()
```

Comment:
- no data when the trip was created.
- no data who created the trip
- no host field
- no image field for trip poster/gallery
- no start location
- no destination location
- no data about when locations will be covered.
- no trip policy data. 
- no detail how many days each trip will take.
- no data about each day.
- no data about the activity / category of this trip